### DISCUSSION TITLE
How to debug the project on windows platform?

### DISCUSSION THREAD

**Discussion by lzh-maker** [OpenRadioss/OpenRadioss#1379] [Q&A]
How to debug the project on windows platform?Can I debug the project with visual studio on windows platform?

**Comment by OlivierWienholtz** [comment]
Hello IZH-Maker

We are currently working on native VS Build and support. We are finalizing it. 
This will permit to debug using Visual Studio.

**Reply by lzh-maker** [reply]
In the x64 Native Tools Command prompt for VS2019, submit the build_ windows. bat script with command parameter  "- arch=win64 debug=1", and then debug based on the generated binary exe file. Do you mean that?   This debugging requires a PDB file. Will a PDB file be generated during the compilation process?

**Reply by OlivierWienholtz** [reply]
Indeed we are finalizing this.

We made batch scripts to have a native build on Windows.
Furthermore, we are working on Visual Studio Integration to have native Visual Studio support. That you can effectively build & debug within Visual Studio. This is in progress.

Options to compile Debug version with Intel Compiler are working. They don't create pdb files but Debugging should be possible.
It has all in the executable like Linux.
